modules/assim.batch/R/pda.define.llik.R

```diff
@@ -98,8 +98,8 @@ pda.calc.error <-function(settings, con, model_out, run.id, inputs, bias.terms){
         # weigh down log-likelihood calculation with neff
         # if we had one beta value (no heteroscadasticity), we could've multiply n_eff*beta
         # now need to multiply every term with n_eff/n 
```

Comment:
@istfer Are these comment lines still in sync with the code after this change?

Reply:
oops, actually no L90-91 is enough and accurate, these could be deleted, I can make a quick PR for that